lib/bracket-matcher.coffee

```diff
@@ -29,6 +29,25 @@ class BracketMatcher
     else
       @pairedCharacters = @defaultPairs
 
+  excludePairs: (excludePairs) ->
```

Comment:
I've refactored the package quite a bit now in an effort to remove duplicated code. This was / is harder than I thought, as it isn't always code that is duplicated, but behavior. I found logic written two different ways between those two files. I believe my rewrite cleans it up a decent amount, and provides for a better separation of concerns.